identity/src/androidTest/java/com/android/identity/UtilTest.java

```diff
@@ -739,6 +739,20 @@ public void testIssuerSignedItemBytesSetValue() {
 
     }
 
+    @Test
+    public void toHexThrows() {
+        assertThrows(NullPointerException.class, () -> Util.toHex((byte[]) null));
+        assertThrows(NullPointerException.class, () -> Util.toHex((byte[]) null));
```

Comment:
Seems to be a duplicated line of code here

Reply:
Good spotting, removed in the next revision.